react/backstop/backstop.js

```diff
@@ -24,25 +24,12 @@ const testComponents = storyBookBackstop.listComponents(dirList);
 // Map discovered Component dirs to Backstop scenarios.
 let scenarios = storyBookBackstop.mapComponents(testComponents, debug);
 
-// The Error and example Template pages need a delay to allow the background
-// animation to complete.
-scenarios = scenarios.map((item) => {
-  const delays = [
-    'Error',
-    'NarrowTemplate',
-    'GenTeaser',
-    'TeaserListing',
-    'Image',
-    'DataTable',
-    'IllustratedHeader'
-  ];
-  // eslint-disable-next-line prefer-const
-  let result = { ...item };
-  if (delays.some((value) => item.label.indexOf(value) > -1)) {
-    result.delay = 5000;
-  }
-  return result;
-});
+// Account for delays necessary to fully render the page.
+// This delay is "jitter" - time the page takes to render AFTER
+// the page has been presented to the user, but before it is visually
+// complete. If we raise this number, it means our performance has gotten
+// worse.
+scenarios = scenarios.map((item) => ({ ...item, delay: 300 }));
```

Comment:
Is this adding a delay of .3 second to all the scenarios instead of just those need a delay?